libs/tup-components/src/tickets/TicketDetailModal/TicketReplyForm.tsx

```diff
@@ -1,47 +1,58 @@
 import React from 'react';
 
-import { Formik, Form, FormikHelpers } from 'formik';
-import {
-  FormikFileInput,
-  FormikTextarea,
-  FormikSelect,
-} from '@tacc/core-wrappers';
-import { FormGroup } from 'reactstrap';
+import { Formik, Form, FormikHelpers, Field } from 'formik';
+import { FormikFileInput, FormikTextarea } from '@tacc/core-wrappers';
+import { FormGroup, Label } from 'reactstrap';
 import { Button, InlineMessage } from '@tacc/core-components';
-import { useTicketReply } from '@tacc/tup-hooks';
-import * as Yup from 'yup';
+import { Ticket, useTicketReply } from '@tacc/tup-hooks';
 import './TicketModal.global.css';
 
 interface TicketReplyFormValues {
   text: string;
   files: File[];
-  status: string;
+  status: boolean;
```

Comment:
**Concern.** Does this implicitly change the value of the data? Consider explicitly changing the value when you need a boolean, rather than changing from string to boolean (here) then back to string later ([lines 52–54](https://github.com/TACC/tup-ui/pull/404/files#diff-17e8589242dc041b374f9548c58c37929d2f421fe10c67b271b97398b8704ea0R52-R54)). I could be missing something. Feel free to explain.

Reply:
I think this is fine as a string actually. I've changed this.